fossunited/chapters/web_template/foss_clubs_home_page/foss_clubs_home_page.html

```diff
@@ -1,273 +1,238 @@
-        }
-
+    #search-club::placeholder {
+      font-size: 1rem;
+    }
 
-    .hero-image img{
-        width: 100%;
+    .btn-primary {
+      width: 100%;
     }
 
+    .hero-image img {
+      width: 100%;
     }
+  }
 </style>
 
-
 <div class="club-hero-section container">
-
-    <div class="hero-container container">
-
-        <div class="hero-header">
-
-            <div class="hero-header-heading">
-                <img src="assets/fossunited/images/chapter/fossclub_logo.svg" alt="">
-                <h1>FOSS <span>Clubs</span></h1>
-            </div>
-
-            <p>{{subtext}}</p>
-
-        </div>
-
-
-        <div class="hero-search">
-
-                <h5>{{subheading}}</h5>
-                <div class="hero-club-search">
-
-                    <input type="text" id="search-club" placeholder="Search by name of club or city">
-
-                </div>
-
-        </div>
-
-        <div class = "hero-cta">
-
-            <div class="button-container">
-
-                    <button class="primary-button" href="{{cta_link}}">{{cta_text}}
-                    </button>
-
-            </div>
-
-        </div>
+  <div class="hero-container container">
+    <div class="hero-header">
+      <div class="hero-header-heading">
+        <img src="assets/fossunited/images/chapter/fossclub_logo.svg" alt="" />
+        <h1>FOSS <span>Clubs</span></h1>
+      </div>
+
+      <p>{{subtext}}</p>
     </div>
 
-    <div class="hero-image">
-        <img src="{{hero_image}}" alt="">
+    <div class="hero-search">
+      <h5>{{subheading}}</h5>
+      <div class="hero-club-search">
+        <input
+          type="text"
+          id="search-club"
+          placeholder="Search by name of club or city"
+        />
+      </div>
     </div>
 
-    <script>
-            function getClubs(query) {
-                return new Promise((resolve, reject) => {
-                    frappe.call({
-                        method: "fossunited.chapters.web_template.foss_clubs_home_page.foss_clubs_home_page.search_foss_club",
-                        args: { query },
-                        callback: (r) => {
-                            resolve(r.message);
-                        }
-                    })
-                })
-            }
-
-            autocomp(document.getElementById("search-club"), {
-                onQuery: async (val) => {
-                    return await getClubs(val);
-                },
-                onSelect: (result) => {
-                    window.location.href = "/" + result.route;
-                    return result.chapter_name;
-                },
-                onRender: (result) => {
-                    const div = document.createElement("div");
-                    div.innerHTML = result.chapter_name;
-                    return div;
-                }
-            });
-
-    </script>
+    <div class="hero-cta">
+
+        {% if cta_link %}
+        <a class="primary-button" href="{{cta_link}}">{{cta_text}} </a>
```

Comment:
@rahulporuri @harshtandiya - This is where the logic is edited, along with the class styles. 

The other changes got in, when I tried a formatting after change. Please note this.